### ISSUE TITLE
Debugger entered--Lisp error: (error "Selecting deleted buffer")

### ISSUE THREAD

**Issue by djangoliv** [syohex/emacs-dired-k#31]
When I open a directory under git with dired, I have this error (with `toggle-debug-on-error`)

```
Debugger entered--Lisp error: (error "Selecting deleted buffer")
  dired-k--parse-git-status("~/workspaces/Proj/trunk/" #<process dired-k-git-status<1>> nil)
  #[514 "\306!\307=\205$
```
Regards

**Comment by syohex** [comment]
I cannot reproduce this issue. Could you tell me how to reproduce this issue in more detail ?

**Comment by djangoliv** [comment]
In fact, I noticed that the problem is only in one of my "git-svn" projects.
I think there is a problem with my .git directory but I don't know which one... 
Sorry for the issue. You can close it.

**Comment by syohex** [comment]
Ah, please report me if it happens again.

**Comment by djangoliv** [comment]
You can try to reproduce it like that:
`git clone https://github.com/NicolasPetton/jade`
open jade directory in dired with this conf:
 ```
(require 'dired-k)
  (setq dired-k-style 'git)
  (setq dired-k-human-readable t)
   (add-hook 'dired-initial-position-hook 'dired-k)
  (add-hook 'dired-after-readin-hook #'dired-k-no-revert)

```

**Comment by syohex** [comment]
I have fixed this issue at #32. Please check latest version.

**Comment by djangoliv** [comment]
Works perfectly.
Thanks
